lighthouse-core/lib/i18n/pre-locale/en-XL.json

```diff
@@ -0,0 +1,3106 @@
+{
```

Comment:
I can maaaaybe see an argument for `pre-locale/en-XL.json`, but definitely don't think we should check in any other `pre-locale/` languages